README.md

```diff
@@ -77,38 +78,73 @@ This sample app lives in this repo in the `sample` directory.
+basis._
+
+The only thing required for Radiography to render composables is to include the tooling library as
+a dependency:
+```kotlin
+dependencies {
+  implementation("androidx.ui:ui-tooling:1.0.0-alphaXY")
+}
+```
+
+### Rendering composables
+
+The `DefaultsNoPii` and `DefaultsIncludingPii` renderers include default Compose renderers – you
+don't need to do anything special. Additional Compose-specific renderers can be found in the
+`ComposableRenderers` object.
 
-Composables can be filtered just like views, but instead of using view ID, you can either specify
-a test tag (as in the string passed to `Modifier.testTag()`), or match on any layout ID (as in
-`Modifier.layoutId()`).
+To create a custom renderer for Compose, implement a `ViewStateRenderer` to handle values of type
+`ComposeView`. The `ComposeView` has a list of all the `Modifier`s that have been applied to the
+composable, including its [semantics](https://developer.android.com/reference/kotlin/androidx/compose/ui/semantics/SemanticsModifier?hl=en#semanticsConfiguration:androidx.compose.ui.semantics.SemanticsConfiguration).
 
+### Filtering composables
+
+Composables can be filtered as well. Compose doesn't have the concept of view IDs, but you
+can explicitly tag composables with strings using the [`testTag`](https://developer.android.com/reference/kotlin/androidx/compose/ui/platform/package-summary#(androidx.compose.ui.Modifier).testTag(kotlin.String))
+modifier, and then tell Radiography to only scan certain tags, or filter certain tags out.
+
+For example, say you have a screen that has a debug drawer, and some body content, like this:
 ```kotlin
-// Filter out views with specific test tags.
-val prettyHierarchy = Radiography.scan(viewFilter = skipTestTagsFilter("debug drawer"))
+@Composable fun App() {
+  ModalDrawerLayout(drawerContent = {
+    DebugDrawer(Modifier.testTag("debug-drawer"))
+  }) {
+    Column {
+      ActionBar()
+      Body(Modifier.testTag("app-body"))
+      BottomBar()
+    }
+  }
+}
+```
 
-// Filter out views with specific layout IDs.
-val prettyHierarchy = Radiography.scan(viewFilter = skipLayoutIdsFilter { layoutId ->
-   layoutId == DebugDrawerLayoutId
-})
+To scan everything _but_ the debug drawer, use `skipTestTagsFilter`:
+```kotlin
+val prettyHierarchy = Radiography.scan(viewFilter = skipTestTagsFilter("debug-drawer"))
 ```
 
-The `DefaultsNoPii` and `DefaultsIncludingPii` renderers already configure the Compose renderers as
-well. Additional compose-specific renderers can be found in the `ComposeLayoutRenderers` object.
-Custom Compose renderers cannot be created at this time, since we're still figuring out exactly
-what the best public API for that looks like. If you need a custom Compose renderer, please file
-an issue on this repo!
+To _only_ scan the contents of the `Body` composable, use `composeTestTagScope`:
+```kotlin
+val prettyHierarchy = Radiography.scan(scanScope = composeTestTagScope("app-body"))
+```
```

Comment:
The example is really cool! One thing that's not obvious when reading this is that at first it sounds like "_only_ scan the contents of the `Body`" is the opposite of "scan everything _but_ the debug drawer" so then we expect some kind of API symmetry which there isn't.

And the main reason for that is that the first is a filter vs the second is a root provider.

Might want to add an extra sentence to the second case, e.g.:

```
To _only_ scan the contents of the `Body` composable, specify a `composeTestTagScope` `ScanScope`  to tell Radiography where to start from:
```

Reply:
Should these be separate samples? I was on the fence whether using the same code for both caused the confusion you pointed out.

Reply:
Split them into separate samples and talked about scoping first.